### DISCUSSION TITLE
Open current file from different commit

### DISCUSSION THREAD

**Discussion by TornaxO7** [NeogitOrg/neogit#341] [Q&A] [answered]
Hello! Is it possible with `neogit` to open up the current opened file but from another commit? I'd like to analyse a file which have been changed from a commit.

**Comment by ttytm** [comment, marked as answer]
Hey @TornaxO7 

as neogit already interacts with diffview would it serve the purpose to use diffview for it?

E.g., you could use `DiffviewFileHistory %`
Or `DiffviewOpen` and specify a revision to open [`DiffviewOpen [git rev] [options] [ -- {paths...}]`](https://github.com/sindrets/diffview.nvim#usage)
